bc_obps/registration/utils.py

```diff
@@ -151,3 +152,17 @@ def paginate_queryset(
             "items": queryset[offset : offset + page_size],
             "count": self._items_count(queryset),
         }  # noqa: E203
+
+
+def is_document_scan_complete(operation: Operation) -> bool:
+    # If we're in the CI or local environment, we don't need to check for document scanning
+    ENVIRONMENT = os.environ.get("ENVIRONMENT")
+    CI = os.environ.get("CI")
```

Comment:
Suggest importing these variables from `from django.conf import settings` and there is a good reason for that: https://docs.djangoproject.com/en/5.2/topics/settings/#using-settings-in-python-code